config/locales/en.yml

```diff
@@ -53,9 +53,24 @@ en:
       scans: 'Scan requests'
   confirmation_email:
     contact:
-      default: "  (650) 723-1493\n  greencirc@stanford.edu"
-    default:
-      from: 'greencirc@stanford.edu'
+      HOOVER:
```

Comment:
Do you think we should promote these into configuration? Without any of the formatting, i18n starts to feel suspect to me..